esmvaltool/diag_scripts/hydrology/derive_evspsblpot.py

```diff
@@ -6,8 +6,8 @@
 American Meteorological Society, 17, 1373-1382, DOI: 10.1175/JHM-D-15-0006.1,
 2016.
 """
-import iris
 import numpy as np
+import iris
```

Comment:
Doesn't isort want it the other way around?

Reply:
Thank you. Running prospector on this file didn't complain about it. This is what pylint says: third party import "import numpy as np" should be placed before "import iris"pylint.